RELEASES.md

```diff
@@ -0,0 +1,69 @@
+# Releases
+
+This document describes the versioning and release process of the OpenFGA Terraform provider. This document will be considered a living document. Scheduled releases, supported timelines, and stability guarantees will be updated here as they change.
+
+## Release Requirements
+OpenFGA endeavours to rapidly release new features to the community.  To support that goal releases are performed based on the impact and quantity of updates to the codebase in a release-ready state.  Releases may be initiated based on a number of factors.
+
+### Functional Changes
+Changes ready for use in production environments which add new functionality or performance improvements may individually trigger an new release.
+
+### Feature Previews
+New features which are ready for testing by the community but are marked as experimental may also trigger a new release.
+
+### Maintenance Updates
+Minor updates, performance improvements, code refinements, or other items which do not introduce signifigant changes may not trigger a release individually, however when several minor changes have been merged a release may be initiated if no new version has been released in two weeks.
+
+### Security Updates and Bugfixes
+Changes made to address a known vulnerability or security issue or fixes to resolve a bug impacting users in production will trigger a release as soon as the changes have been tested and are considered production-ready.  A CVE will also be issued if relevant.
+
+## Versioning
+Releases of the OpenFGA Terraform provider will be versioned using dotted triples. For the purposes of this document, we will refer to the respective components of this triple as \<major\>.\<minor\>.\<patch\>. When release candidates are published, they will include the suffix "-rc" followed by a release candidate number which identifies them as "pre-releases".
+
+### Major and Minor Releases
+Changes in the major and minor version components of the triple indicate changes in the functionality or performance of the OpenFGA Terraform provider which may include breaking changes requiring those integrating the provider in their project to make changes in their use of it when adopting the new version.  It is recommended to fully review the published release notes accompanying a new release before upgrading.
```

Comment:
```suggestion
Changes in the major and minor version components of the triple indicate changes in the functionality or performance of the OpenFGA Terraform provider which may include breaking changes requiring those integrating the provider in their project to make changes in their use of it when adopting the new version.  It is recommended to fully review the published release notes accompanying a new release before upgrading.

While under v1.0.0, you may expect more frequent breaking changes between minor releases. We will aim to avoid breaking changes between patch releases
```